system-parachains/asset-hubs/asset-hub-kusama/src/xcm_config.rs

```diff
@@ -470,6 +478,26 @@ pub type AssetFeeAsExistentialDepositMultiplierFeeCharger = AssetFeeAsExistentia
 	TrustBackedAssetsInstance,
 >;
 
+/// Locations that will not be charged fees in the executor,
+/// either execution or delivery.
+/// We only waive fees for system functions, which these locations represent.
+pub type WaivedLocations = (
+	RelayOrOtherSystemParachains<
+		kusama_runtime_constants::system_parachain::SystemParachains,
```

Comment:
@svyatonik 
this is bug and not working, because `kusama_runtime_constants::system_parachain::SystemParachains` uses `parents: 0` which does not work from parachain - this was fixed by this PR: https://github.com/paritytech/polkadot-sdk/pull/2023  with https://github.com/paritytech/polkadot-sdk/pull/2023/files#diff-a86375df98e04ca3cce1ea35c40257a222e2d5087f5f528ff33307678b78dc2dR535-R553